XenAdmin/Core/Updates.cs

```diff
@@ -442,6 +442,9 @@ public static List<XenServerPatch> RecommendedPatchesForHost(Host host)
                 var minimumPatches = new List<XenServerPatch>();

```

Comment:
Good catch! Let me not fix this here, I have been already working on my next PR that will fix this.